components/SocialLoginGroup/SocialLoginGroup.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { string, func } from 'prop-types';
+import FacebookLogin from 'react-facebook-login/dist/facebook-login-render-props';
+import classNames from 'classnames';
+import GoogleLogin from 'react-google-login';
+import { googleKey, facebookKey } from 'common/config/environment';
+import Alert from 'components/Alert/Alert';
+import styles from './SocialLoginGroup.css';
+
+class SocialLoginGroup extends React.Component {
+  static propTypes = {
+    className: string,
+    onSocialSuccess: func.isRequired,
+  };
+
+  static defaultProps = {
+    className: undefined,
+  };
+
+  state = {
+    errorMessage: '',
+  };
+
+  onFailure = () => {
+    this.setState({ errorMessage: "Couldn't log in with Google" });
```

Comment:
Nice pattern. What about Facebook?

Reply:
The Facebook component only takes the single callback.  Planning on figuring out how/where a facebook login failure comes in.